README.md

```diff
@@ -32,6 +32,20 @@ Percy.logger.error { 'error message' }
 
 Prefer the block form usage `Percy.logger.debug { 'message' }` over `Percy.logger.debug('message')` because it is slightly more efficient when the log will be excluded by the current logging level. For example, if the log level is currently `info`, then a `debug` log in block form will never evaluate or allocate the message string itself.
 
+## Percy::ProcessHelpers
+
+### `gracefully_kill(pid[, grace_period_seconds: 10])`
+
+Returns `true` if the process was successfully killed, or `false` if the process did not exist or its exit status was already collected.
```

Comment:
💯  In Linux processes aren't fully cleaned from the process table until the parent process "collects" or reaps the return code / exit status by waiting on it. The process is considered a [zombie process](https://en.wikipedia.org/wiki/Zombie_process) until the return code is collected and can cause certain kinds of leaks and errors if not collected. In this particular method, we treat both processes that didn't exist and process that were already reaped as the same and just return `false` for both cases, but internally they're actually two different errors that we're handling (see https://github.com/percy/percy-common/pull/2/files#diff-60b93be35329e85abb972b3c700123afR16) — this Errno::ECHILD might be raised if someone first used Ruby's [Process.detach](https://ruby-doc.org/core-2.1.1/Process.html#method-c-detach) on a process, and then calls our `gracefully_kill(pid)` method it.